core-services/prow/02_config/openshift/openldap/_pluginconfig.yaml

```diff
@@ -0,0 +1,13 @@
+approve:
+- commandHelpLink: ""
+  repos:
+  - openshift/openldap
+  require_self_approval: false
+lgtm:
+- repos:
+  - openshift/openldap
+  review_acts_as_lgtm: true
+plugins:
+  openshift/openldap:
+    plugins:
+    - approve
```

Comment:
the rest of our repos are using simply
```
plugins:
  openshift/<repo_name>:
    plugins:
    - approve
```
Do we need/want to add all of this? Also, should we maybe opt this repo out of the BZ process?

Reply:
agreed, we don't need the BZ integration here, this is just used for e2e tests